client/controllers/sourceModal.coffee

```diff
@@ -192,7 +191,11 @@ Template.sourceModal.events
     input.value = @url
     titleInput = instance.find('#title')
     titleInput.value = @subject
+    $(input).trigger('input').trigger('input')
```

Comment:
@nathanathan I went ahead and removed the second trigger method call.
This also fixes [the bug related to the source time not being autofilled](https://www.pivotaltracker.com/story/show/136737137)